gemrb/core/DisplayMessage.cpp

```diff
@@ -321,7 +321,11 @@ void DisplayMessage::DisplayRollStringName(int stridx, unsigned int color, const
 		va_start(numbers, speaker);
 		// fill it out
 		String* str = core->GetString(stridx);
-		vswprintf(tmp, sizeof(tmp)/sizeof(tmp[0]), str->c_str(), numbers);
+#ifndef __amigaos4__
```

Comment:
This should be checking if vswprintf is available instead, in case some other platform suffers from it. See how we achieve this via CHECK_FUNCTION_EXISTS in cmake.

Reply:
I also don't understand why this is needed in the end — aren't you adding a fallback implementation in the helpers?

Reply:
Uhm.. this could come from something old before i've added them in OS4Helpers. i'll remove it. Sorry